board/bootstub.c

```diff
@@ -72,11 +68,12 @@ extern void *_app_start[];
 // BOUNTY: $200 coupon on shop.comma.ai or $100 check.
 
 int main(void) {
+  clock_init();
```

Comment:
Sure it's fine to change the order of this? Any reason? We had some issues with the pedal code a few weeks ago which were due to having interrupts enabled while this runs or something iirc.
If not needed, I'd like to keep this unchanged for now

Reply:
Think I will keep it as is for now and will change later after checks and more reading, but think clocks should always go first(power, core clock, buses etc)

Reply:
I;m really reluctant to changing this without proper testing (should be a separate PR too imo), so let's try to keep all the functional stuff the same in this PR. This should purely be a refactor without any functional changes